### PR TITLE
Add review comments to mail notifications

### PR DESCRIPTION
**Pull request by guillep2k**
As title. Example e-mail from a review:

On GMail:

image

On stupid Outlook 2010:

image

### CODE REVIEW

File: models/issue_comment.go [hunk 1]
```diff
@@ -1016,10 +1016,6 @@ func fetchCodeCommentsByReview(e Engine, issue *Issue, currentUser *User, review
 		return nil, err
 	}
 
-	if err := CommentList(comments).loadPosters(e); err != nil {
```
Comment (zeripath) [inline models/issue_comment.go:1019]: Is this deliberate?

Comment (guillep2k) [reply models/issue_comment.go:1019]: Yes, it's a duplicate of the same action two lines below (or was it above?)

**Review by lunny — APPROVED** [review]

**Comment by guillep2k** [comment]
This needs update after merging #8995

**Comment by guillep2k** [comment]
>
> ~This needs update after merging #8995~

EDIT: It's not the case. It's alright as it is.

**Review by zeripath — APPROVED** [review]

**Comment by zeripath** [comment]
make lg-tm work

**Pull request merged**